packages/clerk-js/src/ui/elements/Tabs.tsx

```diff
@@ -184,7 +185,6 @@ export const TabPanel = (props: TabPanelProps) => {
     <Flex
       id={`cl-tabpanel-${tabIndex}`}
       role='tabpanel'
-      tabIndex={0}
```

Comment:
@desiprisg accessibility wise you need tabindex here as well

https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-automatic

Reply:
You're right. We probably should remove the `focusRing` from the `div` though.

Reply:
Hey peeps, is this one ready to get merged?